README.md

```diff
@@ -47,12 +59,12 @@ app.whoami(function (err, data) {
 //  - feed.add appends a message to your key's chain.
 //  - the `type` attribute is required.
 
-app.publish({ type: 'post', text: 'My First Post!' }, function (err, msg, hash) {
+app.publish({ type: 'post', text: 'My First Post!' }, function (err, msg) {
   // the message as it appears in the database:
   console.log(msg)
 
   // and its hash:
-  console.log(hash)
+  console.log(msg.key)
 })
 
 // stream all messages for all keypairs.
```

Comment:
In the two `pull` examples below... `pull.collect` is used ... I wonder about adding a quick note here about what pull.collect does. 

They both say "// stream all ..." but the what the function does is collect the entire thing into an array, calls back, and then ends.
Also could rename `ary` to `messagesArray`
could throw a link to this too...
https://github.com/pull-stream/pull-stream/blob/master/docs/sinks/collect.md

Reply:
Changed in 3302fcf
Its ok?

Reply:
yep!